docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+# Tizen Window System Shell
+
+
+The Tizen window system shell (TZSH) is an interface for manipulating windows of the sytem graphical user interfaces (GUI) services such as the quickpanel and the softkey. It provides C libraries that allow developers to control windows of system GUI services and be notified state changes of them within an application.
+
+Since the system GUI services are different processes from the application, all of operations of the TZSH are performed using the inter-process communication (IPC) through the display server. Below figure shows a simple S/W architecture of the TZSH.
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+The TZSH provides following libraries to support each system GUI services in the Tizen:
+
+- tzsh-quickpanel: For the quickpanel window which shows notifications and system setup widgets.
+
+- tzsh-softkey: For the softkey window which shows the device back and home buttons.
+
+Most applications don't need to use the TZSH's functionalities. However, some applications may want to perform manipulation of system GUI service's window in certain situations.  For example, media player application needs to close the quickpanel window during playback of video. The tzsh-quickpanel can be used in this case.
```

Comment:
Not all applications require to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service's window. For example, media player application needs to close the quick-panel window during playback of video. In this case, you can use the tzsh-quickpanel library.

Reply:
Thanks I will change as your comment except 'quick-panel'.